web_src/css/features/colorpicker.css

```diff
@@ -0,0 +1,159 @@
+/* This is a stripped-down version of coloris's CSS tailored to our needs. It does only include
+   opaqua colors, and if more features like opacity are needed, the CSS needs to be extended
+   based on upstream: https://github.com/mdbassit/Coloris/blob/main/src/coloris.css. */
+
+input.color-picker {
+  padding-top: 8px !important;
+  padding-bottom: 8px !important;
+  padding-left: 32px !important;
+}
+
+.clr-picker {
```

Comment:
Some confusing names appear together?

`colorpicker.js` vs `.color-picker` vs `.clr-picker`

Reply:
Yes, but we don't control the `.clr-picker` unfortunately.

Reply:
Could others `colorpicker.js` / `.color-picker`/ `.color.picker` be clarified?

Reply:
Definitely. The `.color-picker` I will surely replace and then name the child elements accordingly.

Reply:
@wxiaoguang done, now there is a single root `js-color-picker-input` element and both `.color.picker` and `.color-picker` are removed.